src/objects/Attendance_Event__c.object

```diff
@@ -0,0 +1,196 @@
+                    <default>false</default>
+                    <label>Class Tardy</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Contact__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The student who is either tardy or absent.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The student who is either tardy or absent.</inlineHelpText>
+        <label>Contact</label>
+        <referenceTo>Contact</referenceTo>
+        <relationshipLabel>Attendance Events</relationshipLabel>
+        <relationshipName>Attendance_Events</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Course_Connection__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The class and period in which the student was either absent or tardy.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The class and period in which the student was either absent or tardy.</inlineHelpText>
+        <label>Course Connection</label>
+        <referenceTo>Course_Enrollment__c</referenceTo>
+        <relationshipLabel>Attendance Events</relationshipLabel>
+        <relationshipName>Attendance_Events</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Date__c</fullName>
+        <description>The date when this event occurred.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The date when this event occurred.</inlineHelpText>
+        <label>Date</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Date</type>
+    </fields>
+    <fields>
+        <fullName>End_Time__c</fullName>
+        <description>For pre-excused absences, the time when this absence will end.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>For pre-excused absences, the time when this absence will end.</inlineHelpText>
+        <label>End Time</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Time</type>
+    </fields>
+    <fields>
+        <fullName>Reason__c</fullName>
+        <description>The reason for the tardy/absence.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The reason for the tardy/absence.</inlineHelpText>
+        <label>Reason</label>
+        <length>32768</length>
+        <trackTrending>false</trackTrending>
+        <type>LongTextArea</type>
+        <visibleLines>3</visibleLines>
+    </fields>
+    <fields>
+        <fullName>Start_Time__c</fullName>
+        <description>The time when the pre-excused absence will start.</description>
```

Comment:
May want to change the description here a bit to match the description for end time "For pre-excused absences..."

Reply:
Should be good now! @lifewithryan